=========================================================================================================================

# How to determine the horizon of the visible universe in a Jupyter Notebook

=========================================================================================================================

## 20 correlations establishing the theoritical value of the horizon of the visible universe see [Observable_universe](https://en.wikipedia.org/wiki/Observable_universe) and [Schwarzschild_radius](https://en.wikipedia.org/wiki/Schwarzschild_radius)

# with ± 13.811 billion (10^9) years approximation

=========================================================================================================================

List of physical (NIST CODATA2018) and mathematical constants used:

* $\pi=3.141592653589793$
* Euler natural logarithm $e=2.718281828459045$
* Euler Mascheroni $\gamma=0.5772156649015329$
* Apery Constant $\xi(3)=1.202056903159594$ [Apery Constant](https://en.wikipedia.org/wiki/Ap%C3%A9ry%27s_constant)
* Atiyah's $\Gamma=25.178097241906$ [Michael_Atiyah](https://en.wikipedia.org/wiki/Michael_Atiyah)
* Lucas Mersenne number [Edouard Lucas](https://en.wikipedia.org/wiki/%C3%89douard_Lucas)
* OM Monster [Monster_group](https://en.wikipedia.org/wiki/Monster_group) also known as Fischer–Griess monster
* OB Baby Monster [Baby_monster_group](https://en.wikipedia.org/wiki/Baby_monster_group) baby monster group B
* OP Pariah Group [Pariah_group](https://en.wikipedia.org/wiki/Pariah_group) six sporadic simple groups
* Eddington Electric Constant $a=137.0359990836958$ also known as the inversed fine structure constant CODATA2018
* $c=299792458.0$ m/s CODATA2018
* $h=6.62607015.10^{-34}$ $J.Hz^{-1}$ Planck constant CODATA2018
* $\hbar=1.0545718176461565.10^{-34}$ $J.s$ Planck constant over 2 pi CODATA2018
* $l_P=1.616255.10^{-35}$ m Planck length CODATA2018
* $m_P=2.176434.10^{-8}$ kg Planck mass CODATA2018
* ƛ$_e$ $=3.861 592 6796.10^{-13}$m Reduced (Electron) Compton Wavelength CODATA2018
* ƛ$_p$ $=2.103 089 103 36.10^{-16}$m Reduced (Proton) Compton Wavelength CODATA2018
* $\lambda_{wi}=2.897771955.10^{-3}$ m K (exact) Wien wavelength displacement law constant CODATA2018
* $k_B=1.380 649.10^{-23}$ J K^-1  (exact) Boltzmann constant CODATA2018
* $t_K=9600.60(1)$s non-Doppler Kotov Periodicity (1000 ppb)
* Fermi coupling constant : $1.1663787.10^{-5}$    (GeV)-2          (51 ppb)
* Fermi's ratio   F = Fermi's mass / m_e = 573007.3625        (25 ppb)
* Fermi-Atiyah's ratio : $F_A = (2\Gamma . 137)^{3/2}$ ≈ 573007.3652        (0.22 ppb)
* Mass of the electron $m_e=9.1093837015.10^{-31}$ kg CODATA2018
* Mass of the proton $m_p=1.672 621 923 69.10^{-27}$ kg CODATA2018
* Magnetic Moment of the electron/Bohr magneton: $d_e$ ≈ 1.00115965218128 (2 × 10-4  ppb)
* Magnetic moment anomaly $1159.65218091.10^{−6}$ ± 0.00000026 [Particle Data Group Leptons](http://pdg.lbl.gov/2019/tables/rpp2019-sum-leptons.pdf) M. Tanabashi et al. (Particle Data Group), Phys. Rev. D 98, 030001 (2018) and 2019
* Boson $W=80.379 GeV$ ± 0.012 [Particle Data Group Bosons](http://pdg.lbl.gov/2019/tables/rpp2019-sum-gauge-higgs-bosons.pdf) M. Tanabashi et al. (Particle Data Group), Phys. Rev. D 98, 030001 (2018) and 2019
* Boson $Z=91.1876 GeV$ ± 0.0023 [Particle Data Group Bosons](http://pdg.lbl.gov/2019/tables/rpp2019-sum-gauge-higgs-bosons.pdf) M. Tanabashi et al. (Particle Data Group), Phys. Rev. D 98, 030001 (2018) and 2019
* Lepton $e=0.5109989461 MeV$ [Particle Data Group Leptons](http://pdg.lbl.gov/2019/tables/rpp2019-sum-leptons.pdf) M. Tanabashi et al. (Particle Data Group), Phys. Rev. D 98, 030001 (2018) and 2019
* Baryon $p=938.272081 MeV$ [Particle Data Group Baryons](http://pdg.lbl.gov/2019/tables/rpp2019-sum-baryons.pdf) M. Tanabashi et al. (Particle Data Group), Phys. Rev. D 98, 030001 (2018) and 2019
* Baryon $n=939.565413 MeV$ [Particle Data Group Baryons](http://pdg.lbl.gov/2019/tables/rpp2019-sum-baryons.pdf) M. Tanabashi et al. (Particle Data Group), Phys. Rev. D 98, 030001 (2018) and 2019
* $G=6.6743.10^{-11}$ $m^{3}.kg^{-1}.s^{-2}$ Newtonian constant of gravitation CODATA2018
* $G_q=6.6755.10^{-11}$ $m^{3}.kg^{-1}.s^{-2}$ Newtonian constant of gravitation measured by T.Quinn et al. (2013) BIPM Sevres [Improved determination of G using two methods](https://www.bipm.org/utils/en/pdf/PhysRevLett.111.101102.pdf)
* $G_s=6.67545372.10^{-11}$ $m^{3}.kg^{-1}.s^{-2}$ Newtonian constant of gravitation estimate by Francis M. Sanchez (Jan 2020)
* Galaxies Doppler radius $R=c.f /\Delta.f$=Universe Schwarzschild radius: $2G.M/c^2$
* Hubble-Lemaître radius: $c/H_0$

𝛢𝛼 𝛣𝛽 𝛤𝛾 𝛥𝛿 𝛦𝜀𝜖 𝛧𝜁 𝛨𝜂 𝛩𝜃𝜗 𝛪𝜄 𝛫𝜅 𝛬𝜆 𝛭𝜇 𝛮𝜈 𝛯𝜉 𝛰𝜊 𝛱𝜋 𝛲𝜌 𝛴𝜎 𝛵𝜏 𝛶𝜐 𝛷𝜙𝜑 𝛸𝜒 𝛹𝜓 𝛺𝜔

In [2]:
from scipy import constants
import numpy as np

In [12]:
c=constants.c
print(c)
h=constants.h
print(h)
hbar=constants.hbar
print(hbar)
a=137.0359990836958
print(a)
d_e=1.00115965218128
print(d_e)
xi3=1.202056903159594
print(xi3)
xi2=np.pi**2/6
print(xi2)
#print(c,h,hbar,a,d_e,xi3,xi2)
e=np.e
print(e)
gamma=np.euler_gamma
print(gamma)
Gamma=(constants.fine_structure**-1*gamma)/np.pi
print(Gamma)
# F**(2/3)=2*137*Gamma
# Fermi's mass given by 
F=(2*137*Gamma)**1.5
print(F)
lambdabare=constants.hbar/(constants.m_e*constants.c)
print(lambdabare)
lambdabar_F=lambdabare/F
print(lambdabar_F)
lambdabar_p=(constants.hbar/(constants.m_p*constants.c))
print(lambdabar_p)
H=1837.152646   # (0.06 ppb)
p=1836.15267343 # 1836.1526734400013 #(0.06 ppb)
n=1838.68366173 # 1838.6836617324586 #(0.5 ppb)
print(H,p,n)
m_e=constants.m_e
m_p=constants.m_p
m_n=constants.m_n
print(m_e,m_p,m_n)
beta=(H-p)**(-1)
print(beta)
j=(8*np.pi**2)/np.log(2)
print(j)
k_B=constants.Boltzmann
T_cmb=2.72582
print(k_B,T_cmb)
lambdabar_cmb=(constants.hbar*constants.c)/(k_B*T_cmb)
print(lambdabar_cmb)
# Planck's length CODATA2018 (hbar.G/c^3)^1/2 = 1.61639471.10^-35
# l_P=1.616255*10**(-35) CODATA2018
G_s=6.67545372*10**-11
print(G_s)
l_P=((constants.hbar*G_s)/constants.c**3)**0.5
print(l_P)
ƛ_w=(constants.hbar/(constants.m_e*constants.c))/F**2
print(ƛ_w)
lambdabar_w=(constants.hbar/(constants.m_e*constants.c))/F**2
#2.897 771 955... e-3 m.K - NIST CODATA 2018
b=2.897771955*10**(-3)
#lambdawien=(h*c)/(k_B*)
# b=lambda/lambdawien
print(b)
𝜔=(h*c)/(k_B*b)
print(𝜔)
l_ph=(lambdabare*(np.pi*(a**2))**2)
print(l_ph)
aprime=137.1106292
# Hydrogen Atom Radius
r_H=(lambdabare*aprime)
print(r_H)
# Hydrogen Atom Radius
r_H=(lambdabare*aprime)
print(r_H)
Lucas=(2**127)-1
print(Lucas)
OM=2**46 * 3**20 * 5**9 * 7**6 * 11**2 * 13**3 * 17 * 19 * 23 * 29 * 31 * 41 * 47 * 59 * 71 # 808017424794512875886459904961710757005754368000000000
print(OM)
OB=2**41 * 3**13 * 5**6 * 7**2 * 11 * 13 * 17 * 19 * 23 * 31 * 47 # 4154781481226426191177580544000000
print(OB)
OP=2**8 * 3**7 * 5**6 * 7 * 11 * 31 * 37 * 67 * 2**9 * 3**4 * 5 * 7**3 * 11 * 19 * 31 * 2**14 * 3**3 * 5**3 * 7 * 13 * 29 * 2**21 * 3**3 * 5 * 7 * 11**3 * 23 * 29 * 31 * 37 * 43 * 2**7 * 3**5 * 5 * 17 * 19 * 2**3 * 3 * 5 * 7 * 11 * 19
print(OP)

299792458.0
6.62607015e-34
1.0545718176461565e-34
137.0359990836958
1.00115965218128
1.202056903159594
1.6449340668482264
2.718281828459045
0.5772156649015329
25.178097241906
573007.364905975
3.8615926796089057e-13
6.739167620022749e-19
2.1030891033555923e-16
1837.152646 1836.15267343 1838.68366173
9.1093837015e-31 1.67262192369e-27 1.67492749804e-27
1.000027430752471
113.91063459990004
1.380649e-23 2.72582
0.0008400718024707187
6.67545372e-11
1.6163947113699356e-35
1.1761048867371158e-24
0.002897771955
4.965114232061555
0.0013440173951173593
5.294654020152911e-11
5.294654020152911e-11
170141183460469231731687303715884105727
808017424794512875886459904961710757005754368000000000
4154781481226426191177580544000000
2663849798329448283764291471531459318169741293570162121768960000000000000


In [6]:
# URL= 'https://github.com/laguer/hubble-table/hubble-table.csv'
URL= 'https://raw.githubusercontent.com/LaGuer/hubble-table/master/hubble-table.csv'
from urllib.request import urlretrieve
urlretrieve (URL, 'hubble-table.csv')

('hubble-table.csv', <http.client.HTTPMessage at 0x7f1910261e90>)

In [20]:
URL= 'https://raw.githubusercontent.com/LaGuer/hubble-table/master/constant-table.csv'
from urllib.request import urlretrieve
urlretrieve (URL, 'constant-table.csv')

('constant-table.csv', <http.client.HTTPMessage at 0x7f190e1df410>)

In [21]:
import pandas as pd
#from StringIO import StringIO
#df = pd.read_csv('hubble-table.csv')
df = pd.read_csv("hubble-table.csv", dtype = {"gly_value" : "float64","m_value" : "float64"}, skiprows = [1], sep = ",")
#df1 = pd.read_csv("constant-table.csv", dtype = {" c_value" : "float64"}, skiprows = [1], sep = ",")
#df = pd.to_csv('hubble-table.csv', float_format='%.3f')
df.head ()
#df1.head ()

Formula     gly_value  \
0           ƛe.exp((e 4e-1/a-ln2(P**4/a**3))/2)**1/2  1.381170e+10   
1                                ƛFe(Pe**e)**2/√(pn)  1.381260e+10   
2             (ƛe**2/l P)(j/16)**16( e**e )**2.d_e√2  1.381200e+10   
3  ((a/√137(4πF)**2)ƛe4 l_ph**3(ƛ_CMB/l_P)^8 )**(...  1.381189e+10   
4                               2βƛej**17(4π)**2√137  1.381198e+10   

        m_value  
0   1.30*10**26  
1   1.30*10**26  
2   1.30*10**26  
3   1.30*10**26  
4   1.30*10**26

In [23]:
df1 = pd.read_csv("constant-table.csv", dtype = {"value" : "float64"}, skiprows = [2], sep = ",")
#df = pd.to_csv('hubble-table.csv', float_format='%.3f')
#df1.head ()
df1.tail ()

symbol                                              value
32    G_s                                     6.67545372e-11
33  Lucas            170141183460469231731687303715884105727
34     OM  8080174247945128758864599049617107570057543680...
35     OB                 4154781481226426191177580544000000
36     OP  2663849798329448283764291471531459318169741293...

In [24]:
df1.dtypes

symbol    object
 value    object
dtype: object

In [736]:
# 3**210

In [737]:
# Using G_s the Newtonian constant of gravitation estimate by Francis M. Sanchez (Jan 2020)
G_s=6.67545372*10**-11
print(G_s)

6.67545372e-11


=========================================================================================================================

# Universe Horizon Radius calculus in 3 minutes (method 1)
$$R_U=\frac{2.\hbar^2}{G.m_e.m_p.m_n}$$

In [739]:
R_U=2*constants.hbar**2/(G_s*constants.m_e*constants.m_p*constants.m_n)
print(R_U)

1.3056258355517771e+26


## Yields 13.802861522337478 Gly (G) and 13.80047597102314 Gyr (G_s)

In [740]:
R_U/(constants.light_year*10**9)

13.80047597102314

=========================================================================================================================

# Nambu Universe Radius calculus in 3 minutes (method 2)
$$R_N=\frac{2.\hbar^2}{G.(m_N)^3}$$

### https://en.wikipedia.org/wiki/Yoichiro_Nambu

With Nambu mass $$m_N=m_e.a$$ and $$a=1/\alpha$$

In [741]:
a=(1/constants.fine_structure)
print(a)

137.0359990836958


In [742]:
m_N=constants.m_e*a
print(m_N)

1.2483134965717876e-28


In [743]:
R_N=2*constants.hbar**2/(G_s*((a)*constants.m_e)**3)
print(R_N)

1.7128941663434102e+26


## One obtains 18.10530562420883 Gly with method 2

In [744]:
R_N/(constants.light_year*10**9)

18.10530562420883

$$R_N'=ƛ_e(2R/R_N)^{210}$$ and $$R=\frac{2.\hbar^2}{G.m_e.m_p.m_H}$$

$$H=p+1-\frac{1/2}{a'^2}$$ and $a'=H/p=137.11$

In [745]:
# proton mass 1.672 621 923 69 e-27
constants.m_p

1.67262192369e-27

In [746]:
H=constants.m_e*1837.152645
print(H)

1.6735328361530618e-27


In [747]:
R=2*constants.hbar**2/(G_s*constants.m_e*constants.m_p*H)
print(R)

1.3067138970179814e+26


In [748]:
R_Nprime=(constants.hbar/(constants.m_e*constants.c))*(2*R/R_N)**210
print(R_Nprime)

1.3108749648748815e+26


## One obtains 13.855959311747384 Gly with method 2'

In [749]:
R_Nprime/(constants.light_year*10**9)

13.855959311747384

=========================================================================================================================

# Eddington Universe Radius calculus in 3 minutes (method 3)
$$R_{Edd}=\frac{2.\hbar^2}{G.m_e.m_p^2}$$

### https://en.wikipedia.org/wiki/Eddington_number

In [750]:
# Number of Hydrogen Atoms of mass m_H in the visible universe
N_Edd=136*2**256
print(N_Edd)

15747724136275002577605653961181555468044717914527116709366231425076185631031296


In [751]:
m_H=1837.152645*constants.m_e
print(m_H)

1.6735328361530618e-27


In [752]:
20/3

6.666666666666667

In [753]:
R_Edd=(6.666666666666667*N_Edd*G_s*m_H)/constants.c**2

## One obtains 13.793532128031092 Gly for R_Edd

In [754]:
R_Edd/(constants.light_year*10**9)

13.793532128031092

In [755]:
R_Edd2=2*constants.hbar**2/(G_s*constants.m_e*constants.m_p**2)
print(R_Edd)

1.3049688972818604e+26


## One obtains 13.819498813523245 Gly for R_Edd2

In [756]:
R_Edd2/(constants.light_year*10**9)

13.819498813523245

=========================================================================================================================

# Francis M. Sanchez Universe Radius calculus in 3 minutes (method 4)
$$R_{Francis.M.Sanchez}=ƛ_e.g(6)$$

With $$g(k)=exp(2^{k+1/2})/k$$ when $$k=6$$

and ƛ$_e=\frac{\hbar}{m_e.c}$ as the reduced Compton (electron) wavelength

In [757]:
# Reduced Compton Wavelength
(constants.hbar/(constants.m_e*constants.c))

3.8615926796089057e-13

In [758]:
import numpy as np
from scipy import constants
R_FMS=(constants.hbar/(constants.m_e*constants.c))*np.exp(2**(6+(1/2)))/6
print(R_FMS)

1.307568892158173e+26


## One obtains 13.821014095558315 Gly with method 4

In [759]:
R_FMS/(constants.light_year*10**9)

13.821014095558315

=========================================================================================================================

# Universe Radius calculus based on Rydberg reduced wavelength (method 5) 
$$R_{Rydberg}=\frac{(ƛ_{Ryd}.n^4)^2}{ƛ_p}$$

In [760]:
a=constants.fine_structure**-1
print(a)

137.0359990836958


In [761]:
a**2

18778.865044866678

In [762]:
# n equals mass ratio between Neutron and Electron
n=(constants.m_n)/(constants.m_e)
print(n)

1838.6836617324586


In [763]:
n**4

11429522003521.688

In [764]:
lambdabarRyd=2*(constants.hbar/(constants.m_e*constants.c))*(a**2)
print(lambdabarRyd)

1.4503265557724145e-08


In [765]:
lambdabar_p=(constants.hbar/(constants.m_p*constants.c))
print(lambdabar_p)

2.1030891033555923e-16


In [766]:
R_Ryd=((lambdabarRyd*n**4)**2)/lambdabar_p
print(R_Ryd)

1.3065621143319103e+26


## One obtains 13.810372445537942 Gly with method 5

In [767]:
# Incorrect result , method to be modified
R_Ryd/(constants.light_year*10**9)

13.810372445537942

=========================================================================================================================

# Universe Radius calculus based on pi (method 6) 
$$R_{\pi}=ƛ_e.\pi^{155/2}$$

In [768]:
R_pi=(constants.hbar/(constants.m_e*constants.c))*constants.pi**(155/2)
print(R_pi)

1.3058148293550039e+26


## One obtains 13.802473637099501 Gly with method 6

In [769]:
R_pi/(constants.light_year*10**9)

13.802473637099501

=========================================================================================================================

# Universe Radius calculus based on 2 (method 7) 
$$R_{2}=ƛ_e.2^{128}$$

In [770]:
R_2=(constants.hbar/(constants.m_e*constants.c))*2**128
print(R_2)

1.3140318971018876e+26


## One obtains 13.889328111716429 Gly with method 7

In [771]:
R_2/(constants.light_year*10**9)

13.889328111716429

=========================================================================================================================

# Universe Radius calculus based on Fermi's mass and Kotov period (method 8) 

$$R_{kotov-fermi}=\frac{2.(c.t_k)^2/ƛ_e}{F^2}$$

Fermi's mass given by the relation: $$F=(2\times 137 \times \Gamma)^{3/2}$$

In [772]:
# Atiyah's Gamma
Gamma=a*np.euler_gamma/np.pi
print(Gamma)

25.178097241906


In [773]:
# F**(2/3)=2*137*Gamma
# Fermi's mass given by 
F=(2*137*Gamma)**1.5
print(F)

573007.364905975


In [774]:
((2*(constants.c*9600.6)**2)/(constants.hbar/(constants.m_e*constants.c)))/F**2

1.3067162206121416e+26

In [775]:
R_kf=(2*(constants.c*9600.6)**2)/(constants.hbar/(constants.m_e*constants.c)) / F**2
print(R_kf)

1.3067162206121416e+26


https://en.wikipedia.org/wiki/Light-year

In [776]:
# https://en.wikipedia.org/wiki/Light-year
constants.light_year

9460730472580800.0

In [777]:
R_kf/(constants.light_year*10**9)

13.812001350205271

=========================================================================================================================

# Universe Radius calculus based on "a"  (method 9) 
$$a=1/\alpha$$ $\alpha$ being the fine structure constant  $$R_{a}=ƛ_e.(2.\pi^2.a^3)^5$$

In [778]:
lambdabare=(constants.hbar/(constants.m_e*constants.c))
print(lambdabare)

3.8615926796089057e-13


In [779]:
(2*constants.pi**2)*((1/constants.fine_structure)**3)**5

2.2276645699260076e+33

In [780]:
R_a=((constants.pi**2*(2*(1/constants.fine_structure)**3))**5)*(constants.hbar/(constants.m_e*constants.c))
print(R_a)

1.3059760854303987e+26


In [781]:
R_a/(constants.light_year*10**9)

13.80417811516134

=========================================================================================================================

# Universe Radius calculus based on $\lambda_{CMB}$ (method 10) 


$l_P$ is the Planck length , $j$ is the sternheimer constant

$$R_{cmb} = \frac{(\lambda_{cmb}/(j+1))^2}{l_P}$$

In [782]:
j=(8*np.pi**2)/np.log(2)
print(j)

113.91063459990004


In [783]:
k_B=constants.Boltzmann
T_cmb=2.72582
print(k_B,T_cmb)

1.380649e-23 2.72582


In [784]:
lambdabar_cmb=(constants.hbar*constants.c)/(k_B*T_cmb)
print(lambdabar_cmb)

0.0008400718024707187


In [785]:
lambda_cmb=lambdabar_cmb*2*np.pi
print(lambda_cmb)

0.005278326806259892


In [786]:
# Planck's length CODATA2018 (hbar.G/c^3)^1/2 = 1.61639471.10^-35
# l_P=1.616255*10**(-35) CODATA2018
l_P=((constants.hbar*G_s)/constants.c**3)**0.5
print(l_P)

1.6163947113699356e-35


In [787]:
R_cmb=((lambda_cmb/(j+1))**2)/l_P
print(R_cmb)

1.3053431213288355e+26


In [788]:
R_cmb/(constants.light_year*10**9)

13.79748767932874

=========================================================================================================================

# Universe Radius calculus based on $ƛ_w$, $\gamma$ and $\Gamma$ (method 11) 

$$R_{ƛ_w}=ƛ_w.(\frac{2^{70}.\gamma.n^3}{\Gamma^4})^2$$

$$ƛ_w=ƛ_e/F^2$$

In [789]:
import numpy as np

In [790]:
a=constants.fine_structure**-1
print(a)

137.0359990836958


In [791]:
# Atiyah's Gamma
Gamma=a*np.euler_gamma/np.pi
print(Gamma)

25.178097241906


In [792]:
np.pi

3.141592653589793

In [793]:
ƛ_w=(constants.hbar/(constants.m_e*constants.c))/F**2
print(ƛ_w)

1.1761048867371158e-24


In [794]:
R_w=(((2**70*np.euler_gamma*n**3)/Gamma**4)**2)*ƛ_w
print(R_w)

1.306713911982143e+26


In [795]:
R_w/(constants.light_year*10**9)

13.811976947966931

=========================================================================================================================

# $R =$  ƛe $ g(6)/ (1 + √(137^2+√136)/jn)$ (method 12) 

With $$g(k)=exp(2^{k+1/2})/k$$ when $$k=6$$

and ƛ$_e=\frac{\hbar}{m_e.c}$ as the reduced Compton (electron) wavelength

In [796]:
import numpy as np
from scipy import constants
R_12=((constants.hbar/(constants.m_e*constants.c))*np.exp(2**(6+(1/2)))/6)/(1+np.sqrt(137**2+np.sqrt(136))/(j*n))
print(R_12)

1.3067138950061914e+26


In [797]:
G_12=2*constants.hbar**2/(R_12*m_e*m_p*m_H)
print(G_12)

6.675453730277392e-11


In [798]:
(1+np.sqrt(137**2+np.sqrt(136))/(j*n))

1.0006543109055848

In [799]:
R_12/(constants.light_year*10**9)

13.811976768530982

=========================================================================================================================

# $R =  \frac{2.ƛe(p.n/H^2).g(5)^2}{ln^2 (2 – 1/j.a^2)} $ (method 13) 

With $$g(k)=exp(2^{k+1/2})/k$$ when $$k=5$$

and ƛ$_e=\frac{\hbar}{m_e.c}$ as the reduced Compton (electron) wavelength

In [800]:
import numpy as np
from scipy import constants
# R_13=(2*(constants.hbar/(constants.m_e*constants.c))*(p*n/H**2)*(np.exp(2**(5+(1/2)))/5)**2)/(np.log(2-1/(j*a**2)))**2
# print(R_13)

In [801]:
#R_13/(constants.light_year*10**9)

In [802]:
#(2*(constants.hbar/(constants.m_e*constants.c))*(p*n/H**2)*(np.exp(2**(5+(1/2)))/5)**2)

In [803]:
(np.log(2-1/(j*a**2)))**2

0.480452689883273

In [804]:
#G_13=2*constants.hbar**2/(R_13*m_e*m_p*m_H)
#print(G_13)

=========================================================================================================================

# $R_N(O_M O_B/n_{ph})^2$ (method 14)

In [805]:
n_ph=3.838*10**87
print(n_ph)

3.8379999999999998e+87


In [806]:
R_N

1.7128941663434102e+26

In [807]:
R_14=R_N*(OM*OB/n_ph)**2
print(R_14)

1.3105645508705246e+26


In [808]:
R_14/(constants.light_year*10**9)

13.852678233132401

=========================================================================================================================

# $\lambda_e OM^{7/10}$ (method 15)

In [809]:
lambda_e=lambdabare*2*np.pi

In [810]:
#lambda_e*OM**(7/10)
R_15=lambda_e*OM**(7/10)
print(R_15)

1.3186843114021136e+26


In [811]:
R_15/(constants.light_year*10**9)

13.938504169672099

=========================================================================================================================

# $(2\pi^{32}Pƛe)^2/R_N$

In [812]:
# 
P=lambdabare/l_P
print(P)

2.389015908333496e+22


In [813]:
# Radius in m
R_16=((2*np.pi**32)*P*lambdabare)**2/R_N
print(R_16)

1.3058470789771101e+26


In [814]:
# Radius in Gly
R_16/(constants.light_year*10**9)

13.802814515874134

=========================================================================================================================

# $ƛ_FP^3/p^7$

In [815]:
P

2.389015908333496e+22

In [816]:
R_17=(lambdabar_F*P**3)/p**7
print(R_17)

1.30587575365127e+26


In [817]:
# Radius in Gly
R_17/(constants.light_year*10**9)

13.803117607419157

=========================================================================================================================

# $ƛ_FP^2e/8$

In [818]:
R_18=(lambdabar_F*(P**2)*np.e)/8
print(R_18)

1.3069194966342527e+26


In [819]:
# Radius in Gly
R_18/(constants.light_year*10**9)

13.814149979454358

=========================================================================================================================

# $4P^3l_{Wi}ƛe/R_N$

In [820]:
# l_wi
((2.897771955*10**-3)/2.7255)

0.0010632074683544304

In [821]:
# l_wi=constants.Wien
l_wi=1.0630825*10**-3
print(l_wi)

0.0010630824999999999


In [822]:
R_19=(4*P**3*l_wi*lambdabare)/R_N
print(R_19)

1.3071337184211224e+26


In [823]:
# Radius in Gly
R_19/(constants.light_year*10**9)

13.816414305528234

=========================================================================================================================

# $ƛe.exp((e^{4e-1/a}-ln^2(P^4/a^3))/2)^{1/2}$

In [824]:
(e**(4*e-1/a))

52356.42609346239

In [825]:
P**4/a**3

1.2658205894579623e+83

In [826]:
np.log(P**4/a**3)**2

36614.930925762965

In [827]:
R_20=np.exp(np.sqrt((e**(4*e-1/a)-np.log(P**4/a**3)**2)/2))*lambdabare
print(R_20)

1.3066870223677776e+26


In [828]:
# R_20=(lambdabare*np.exp((e**(4*e-1/a)-np.log(P**4/a**3)**2))/2)**0.5
print(R_20)

1.3066870223677776e+26


In [829]:
# Radius in Gly
R_20/(constants.light_year*10**9)

13.811692724518824

=========================================================================================================================

# $ƛ_F(Pe^e)^2/√(pn)$

In [830]:
lambdabar_f=lambdabare/F
print(lambdabar_f)

6.739167620022749e-19


In [831]:
e*(P*e**e)**2/np.sqrt(p*n)

1.939073444929389e+44

In [832]:
R_21=lambdabar_f*e*(P*e**e)**2/np.sqrt(p*n)
print(R_21)

1.3067740972914103e+26


In [833]:
# Radius in Gly
R_21/(constants.light_year*10**9)

13.812613107187849

In [834]:
a**a/np.pi

2.1189663868972993e+292

=========================================================================================================================

# Measurement Confirmation of the methods given above by:

### Confirmation Measurement #1 (WMAP)
https://en.wikipedia.org/wiki/Wilkinson_Microwave_Anisotropy_Probe

$R_{wmap}=13.74$ ±0.11 Gly (over 9 years)

### Confirmation Measurement #2
https://en.wikipedia.org/wiki/Type_Ia_supernova
Universe Radius measurement based on supernovae 1a
$$R_{SN1a}=13.6(6)Gly$$

### Confirmation Measurement #3 (Planck Space Observatory)
https://en.wikipedia.org/wiki/Planck_(spacecraft)
13.8 billion years (as of 2015) – 13.799±0.021 billion (109) years within the Lambda-CDM concordance model.

$$R_{Planck}=13.799±0.021 Gly$$

### Confirmation Measurement #4 (James Webb Space Telescope)

#### https://en.wikipedia.org/wiki/James_Webb_Space_Telescope

Measurements due to start in 2021

=========================================================================================================================

# Other estimates

## Universe mass calculus given by the relation:

$m_P$ is the Planck Mass , $m_e$ the electron mass, $m_p$ the proton mass, $m_n$ the neutron mass

In [835]:
%%latex
\begin{align}
\frac{m_P^4}{m_e\cdot m_p\cdot\ m_n} = M_U \\
\end{align}

<IPython.core.display.Latex object>

In [836]:
# Planck's mass
m_P=2.176434*10**(-8)
print(m_P)

2.176434e-08


In [837]:
print(m_P**4)

2.2437889614753534e-31


In [838]:
constants.m_e*constants.m_p*constants.m_n

2.5520124761382513e-84

### Universe Mass $8.792233511611562.10^{52}$ kg

In [839]:
M_U=(m_P**4)/(constants.m_e*constants.m_p*constants.m_n)
print(M_U)

8.792233511611562e+52


=========================================================================================================================

## GrandCosmos radius calculus given by the relation:

# $R_{GC} = R_N^2/(2.l_P)$

In [840]:
R_GC=R_N**2/(2*l_P)
print(R_GC)

9.075773400070832e+86


In [841]:
R_GC/(constants.light_year*10**9)

9.59310005329329e+61

=========================================================================================================================

# Volume calculus of R_GC: $V_{GC}=(R_{GC}/r_H)^3(4\pi/3)$

In [842]:
#aprime=a/(H/p)
aprime=137.11
print(aprime)

137.11


In [843]:
# Hydrogen Atom Radius
r_H=(lambdabare*aprime)
print(r_H)

5.2946297230117714e-11


In [844]:
# Volume of R_GC
V_GC=((R_GC/r_H)**3)*(4*np.pi/3)
print(V_GC)

2.109758578455705e+292


In [845]:
a**a/np.pi

2.1189663868972993e+292

In [846]:
from IPython.display import Latex
Latex(r"""\begin{eqnarray}
\ (R_{GC}/r_H)^3(4\pi/3) \approx a^a/\pi \\
\end{eqnarray}""")

<IPython.core.display.Latex object>

In [847]:
aprime=137.1106292

In [848]:
H=p+1 -1/(2*(aprime))**2
print(H)
# p + 1 – 1/2(aH/p)^2


1837.15266013165


In [849]:
# Fermi's Energy in GeV
np.sqrt(1/(1.1663787*10**-5))

292.8061605778735

In [850]:
# ratio of Fermi/Electron (electron in MeV)
(np.sqrt(1/(1.1663787*10**-5))/0.510998950)*1000

573007.3624962899

=========================================================================================================================

$A*OP=1.432938112.10^{365}=\Pi_{Pariah}.a^a=(R_{GC}/l_P)^3$

In [851]:
OP

2663849798329448283764291471531459318169741293570162121768960000000000000

In [852]:
A=a**a
print(A)

6.656929234280262e+292


=========================================================================================================================

$((a/√137(4\pi.F)^2)ƛe^4l{ph}^3(ƛ_{CMB}/l_P)^8)^{1/7}$

# $l_{ph}=(ƛe(\pi a^2)^2)^{-3}  $

In [853]:
lambdabare

3.8615926796089057e-13

In [854]:
# 0.0013440173951173593
l_ph=(lambdabare*(np.pi*(a**2))**2)
print(l_ph)

0.0013440173951173593


In [855]:
l_ph1=(lambdabare*(np.pi*(a**2)**2))**-3
print(l_ph1)

12771295365.93782


In [856]:
l_ph2=(lambdabare*(np.pi*(a**2)**2)**-3)
print(l_ph2)

2.83988529656067e-40


In [857]:
# 𝜉(3)=1.202056903159594
apery=1.202056903159594

In [858]:
# l_ph^-3=16𝜋𝜉(3))/l_{CMB}^3
l_ph3=lambda_cmb/(16*np.pi*apery)**(-1/3)
print(l_ph3)

0.020712279754473314


In [859]:
(lambdabare**4)*(l_ph**3)*(lambdabar_cmb/l_P)**8

2.8736403508066634e+195

In [860]:
a/np.sqrt(137*4*np.pi**2)

1.863350346618644

In [861]:
lambdabare**4

2.223647026545871e-50

In [862]:
l_ph**3

2.4278098495001795e-09

In [863]:
a/(np.sqrt(137)*(4*np.pi)**2)

0.07414035459408844

In [864]:
((a/(np.sqrt(137)*(4*np.pi*F)**2))*(lambdabare**4)*(l_ph**3)*(lambdabar_cmb/l_P)**8)**(1/7)

1.3062424977649562e+26

In [865]:
#((a/np.sqrt(137)*(4*np.pi*F)**2)*(lambdabare**4)*(l_ph**3)*(lambdabar_cmb/l_P)**8)**(1/7)

In [866]:
(a/(np.sqrt(137)*(4*np.pi*F)**2)*(lambdabare**4)*(l_ph**3)*(lambdabar_cmb/l_P)**8)**(1/7)

1.3062424977649562e+26

In [867]:
R_22=((a/(np.sqrt(137)*(4*np.pi*F)**2)*lambdabare**4*l_ph**3)*(lambdabar_cmb/l_P)**8)**(1/7)

In [868]:
# Radius in Gly
R_22/(constants.light_year*10**9)

13.806994095759556

=========================================================================================================================

# $R_{23}=(x.R1^2)/R_N$
* With $x=(11/4)^{3/610}$

In [869]:
((1/np.log(2))**2.5)

2.499978924038854

In [870]:
R1=(np.exp((((a*H)/p*0.6449340668482264)+1)-gamma))*lambdabare
x=(11/4)**(3/610)
RN=1.712894163*10**26 
print(R1,RN,x)

1.4923645070720404e+26 1.7128941630000002e+26 1.0049874827424896


In [871]:
(11/4)**(3/610)

1.0049874827424896

In [872]:
# R_1=1.492365475 × 10^26 
R_23=(x*R1**2)/RN

In [873]:
# Radius in Gly
R_23/(constants.light_year*10**9)

13.811958871705365

=========================================================================================================================

# Monoelectronic universe radius calculus 
$ƛe.exp((\pi^2/6-1).(aH/p) + 1 - \gamma )$
* With $a \approx a.H/p \approx a.((1+1/p)) \approx a.(1/(1/p+1))$


In [874]:
a*(1/(1/p+1))

136.96140756984636

In [875]:
a*((1+1/p))

137.11063122135334

In [876]:
(a*H)/p

137.11063022886907

In [877]:
((np.pi**2)/6)-1

0.6449340668482264

In [878]:
# monoelectronic universe radius in m
R1prime=(np.exp((((a*H)/p*0.6449340668482264)+1)-gamma))*lambdabare
R1=(np.exp(((a*((1+1/p))*0.6449340668482264)+1)-gamma))*lambdabare
print(R1)
print(R1prime)

1.4923654623153317e+26
1.4923645070720404e+26


In [879]:
# Radius in Gly
GlyR1=R1/(constants.light_year*10**9)
GlyR1prime=R1prime/(constants.light_year*10**9)
print(GlyR1)
print(GlyR1prime)

15.774315383368366
15.774305286438809


In [880]:
#3.8610**38  ƛe.exp((\pi^2/6-1).(aH/p) + 1 - \gamma )
(np.exp(((a*H)/p)*((np.pi**(2)/6)-1)+1)-gamma)*lambdabare

2.6580092611335417e+26

$sin^2.\theta$

In [881]:
(1/np.log(2))**(-1/4)

0.9124443057840286

In [882]:
np.log((OM*OB)/2)/(np.log(np.log(2)))/-4

136.9957461328975

=========================================================================================================================

# $2βƛ_e j^{17} (4π)^2 √137$

In [883]:
beta=(H-p)**(-1)
print(beta)

1.0000132985268892


In [884]:
R_j17=(2*beta*lambdabare*j**17*(4*np.pi)**2)*np.sqrt(137)

In [885]:
# Radius in Gly
GlyRj17=R_j17/(constants.light_year*10**9)
GlyR1prime=R1prime/(constants.light_year*10**9)
print(GlyRj17)
print(GlyR1prime)

13.811797040850742
15.774305286438809


# $(ƛ_e^2 /l_P )(j/16)^{16} ( e^e )^2 d_e √2$

In [886]:
l_P

1.6163947113699356e-35

In [887]:
Rj16=(lambdabare**2/l_P*((j/16)**16)*(e**e)**2)*(d_e)*np.sqrt(2)

In [888]:
(j/16)**16

43562081064894.15

In [889]:
(e**e)**2

229.65166408352397

In [890]:
d_e

1.00115965218128

In [891]:
# Radius in Gly
GlyRj17=R_j17/(constants.light_year*10**9)
GlyRj16=Rj16/(constants.light_year*10**9)
print(GlyRj17)
print(GlyRj16)

13.811797040850742
13.812025973801315


# $(ƛ_e3j^j /2H)^{1/6}$

In [892]:
# Radius in m
Rjj=lambdabare*((3*j**j)/(2*H))**(1/6)
print(Rjj)

1.3067153874771553e+26


In [893]:
# Distance Radius Length in Gly
GlyRjj=Rjj/(constants.light_year*10**9)
print(GlyRjj)

13.811992543961518


=========================================================================================================================

# Electric Constant "a" calculus

$a \approx (\frac{\lambda_{cmb}^5.l_{Wien}^4/l_{ph}^9}{\pi^3})^2 \approx (16\xi(3)^3/\omega^4)^2$

In [894]:
a

137.0359990836958

In [895]:
(((lambda_cmb**5*l_wi**4)/l_ph**9)/np.pi**3)**2
#lambda**5

139.09491763634472

In [896]:
#2.897 771 955... e-3 m.K - NIST CODATA 2018
b=2.897771955*10**(-3)
#lambdawien=(h*c)/(k_B*)
# b=lambda/lambdawien
print(b)

0.002897771955


In [897]:
𝜔=(h*c)/(k_B*b)
print(𝜔)

4.965114232061555


In [898]:
(((16*apery)**3)/𝜔**4)**2

137.0364499145383

=========================================================================================================================

# Wien Constant
$ω = hc/k_B \lambda_{wien} = 5(1-e^{-ω} )$
(official value : $hc/bk_B$ = 4.9651141632 : )

In [899]:
#2.897 771 955... e-3 m.K - NIST CODATA 2018
b=2.897771955*10**(-3)
#lambdawien=(h*c)/(k_B*)
# b=lambda/lambdawien
print(b)

0.002897771955


In [900]:
𝜔=(h*c)/(k_B*b)
print(𝜔)

4.965114232061555


In [901]:
𝜆=(h*c)/k_B
print(𝜆)

0.014387768775039337


In [902]:
𝜔1=5*(1-np.e**-(4.9651142317443))
print(𝜔1)

4.965114231744277


=========================================================================================================================

In the following example, one can write the equation with Latex:

In [903]:
from IPython.display import Latex
Latex(r"""\begin{eqnarray}
\ T^2=L^3=M^5=F^7 \\
\ R = \frac{2\cdot \hbar^2}{G.m_e.m_p.m_n} \\
\ M_U = \frac{{R}{c}^2}{2G} = \frac{{m_P}^4}{m_em_pm_n} \\
\frac{r_H}{ƛ_e} = \frac{\sum\frac{1}{n}}{\sum\frac{1}{n^2}}\\
\end{eqnarray}""")


<IPython.core.display.Latex object>

=========================================================================================================================

:star:

# Docs

This project aims at providing a repository dedicated to Dr Francis Michel Sanchez's work in holography and cosmology; it is just a mere continuation of what was initiated and published in a set of theoritical physics articles.


## Publications
* Please, cite this paper (http://vixra.org/pdf/1811.0146v8.pdf) if you employ this package in your research work.
* [Vixra Publications](http://vixra.org/author/francis_m_sanchez)
* [Rxiv Publications](http://rxiv.org/author/f_m_sanchez)
* [Nature: Observations of solar oscillations with periods of 160 minutes](https://www.nature.com/articles/277635a0)
* [A coherent resonant cosmology approach and its implications in microphysics and Biophysics Pages 375-407 of Quantum Systems in Physics, Chemistry](https://www.springer.com/fr/book/9783319502540?token=gbgen&wt_mc=GoogleBooks.GoogleBooks.3.EN#)
* [From Coherent Cosmic Oscillations to the Steady-State Cyclic Ultrafast](https://rxiv.org/abs/1401.0228)
* [Evidence for a Steady-State, Holographic, Tachyonic and Super-Symmetric Cosmology](https://www.researchgate.net/publication/267430152_Evidence_for_a_Steady-State_Holographic_Tachyonic_and_Super-Symmetric_Cosmology)
* [academia.edu Correcting Five Historical Faults in Modern Cosmology](https://www.academia.edu/30843365/Correcting_Five_Historical_Faults_in_Modern_Cosmology)

## Videos

* [L'enigme des grands nombres Part1](https://www.youtube.com/watch?v=ev_eIqLtK8s)
* [L'enigme des grands nombres Part2](https://www.youtube.com/watch?v=i4w392nYqns)
* [L'enigme des grands nombres Part3](https://www.youtube.com/watch?v=b4WvI0yVXnE)
* [Exposés de Francis Sanchez playlist created by J.Bailhache](https://www.youtube.com/playlist?list=PLQXUNrzoAeFbmfHo3jgWzzExlVoBn2JUw)
* [Khelif/Torgoz/Sanchez' Debate on LE FORMALISME EN PHYSIQUE ET EN SCIENCE](https://www.youtube.com/watch?v=TEK2hCI2we0)

* [La science du 3eme millenaire playlist created by J.Bailhache](https://www.youtube.com/playlist?list=PLQXUNrzoAeFbLOuEsltAUWJWCcAEgeGJE)
* [Francis Sanchez Cosmologie 23/03/2017](https://www.youtube.com/watch?v=NmvvIQjxBw8)
* [La complexité mathématique face à la simplicité cosmique - Exposé de Francis Sanchez](https://www.youtube.com/watch?v=dyUlUahgHGg)
* [La méthode scientifique en question et les constantes fondamentales - Exposé](https://www.youtube.com/watch?v=zUt1hxTdiuA)
* [Jean Maruani chez lui et Francis Sanchez autour du nombre 237 29. 12. 17](https://www.youtube.com/watch?v=05g416VWjnw)
* [Pr. Michel Grosmann: Holographie quanto-cosmique](https://www.youtube.com/watch?v=M7bJnekDIt0)
* [Pr. Renée Veysseyre: La constante d'Atiyah réunifie Physique et Mathématique](https://www.youtube.com/watch?v=mbPpKqakDAU)
* [Christian Bizouard, astronome: Omniprésence astrophysique du cycle de Kotov](https://www.youtube.com/watch?v=DGeBC73kKvQ)
* [Pr. Francis Sanchez: Historique, Principe Calculatoire, Axe Topologique et Principe Holique](https://www.youtube.com/watch?v=IgKSrfsqFEs)

## Books

* [Bizouard, Kotov, Sanchez wrote in 20th Natural Philosophy Alliance Proceedings by David de Hilster, published by Lulu.com in 2013]()
* [Quantum Systems in Physics, Chemistry, Springer Ed](https://www.amazon.com/gp/search?index=books&linkCode=qs&keywords=9783319502557)
* [The Worldwide List of Alternative Theories and Critics](https://books.google.com/books?isbn=2902425171)
* [The Dirac Electron: From Quantum Chemistry to Holistic Cosmology by J.Maruani](https://onlinelibrary.wiley.com/doi/abs/10.1002/jccs.201500374#)
* [solar 22 years cycle](https://link.springer.com/article/10.1007%2Fs10509-016-2985-8)
* [Current Issues in Cosmology](https://www.cambridge.org/gb/academic/subjects/physics/cosmology-relativity-and-gravitation/current-issues-cosmology?format=HB&isbn=9780521858984)
* ```@inproceedings{Sanchez2015RemarkablePO,
  title=Remarkable Properties of the Eddington Number 137 and Electric Parameter 137 . 036 excluding the Multiverse Hypothesis},
  author={Francis M. Sanchez and Valery A. Kotov,
  year=2015```

## Papers

* [College de France (2004): Discussions sur les oscillations cosmiques, les nombres sans dimension et les periodicites en microphysique et cosmologie - College de France 27th Feb 2004](https://lempel.pagesperso-orange.fr/occ_cdf.pdf)

More to come soon stay tuned.../.:stars:.

=========================================================================================================================

In [904]:
#Write your equation in a dedicated cell with latex

from IPython.display import Latex
Latex(r"""\begin{eqnarray}
T^2=L^3=M^5=F^7 \\
\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M \\
\frac{\pi_Pt}{\pi_Bb} = 6 \cdot \frac{\pi^5}{p_H^9} \\
\end{eqnarray}""")

<IPython.core.display.Latex object>

In [905]:
#Write Bekenstein-Hawking Entropy 
from IPython.display import Latex
Latex(r"""\begin{eqnarray}
S = \frac{c^3 k A}{4 \hbar G} \\
\end{eqnarray}""")

<IPython.core.display.Latex object>